sklearn/impute/_base.py

```diff
@@ -598,7 +598,7 @@ def fit(self, X, y=None):
         Returns
         -------
         self : object
```

Comment:
This is no longer accurate

Reply:
Do you mean the return comment of `_fit` or `fit`?